src/status_im2/navigation/core.cljs

```diff
@@ -95,8 +96,24 @@
                                     (options/merge-top-bar (options/topbar-options) options)
                                     {:topBar {:visible false}}))}})))
 
+;; NAVIGATE-TO-WITHIN-STACK
+(defn navigate-to-within-stack
+  [[comp comp-id]]
+  (let [{:keys [options]} (get views/screens comp)]
+    (navigation/push
+     (name comp-id)
```

Comment:
Pushing screens inside different stacks is useful when we have a parallel navigation system. Something like when we had bottom tabs, where all bottom tabs had their stacks etc.
For vertical navigation, I am not sure if this will change anything. Pushing a new stack on the root, or on the top of another stack should have the same effect.